ui/src/app/controllers/views/ControllerDetails/ControllerSummary/ServicesCard/ServiceStatus/ServiceStatus.tsx

```diff
@@ -0,0 +1,36 @@
+import { Icon } from "@canonical/react-components";
+
+import type { Service } from "app/store/service/types";
+import { ServiceStatus as ServiceStatusType } from "app/store/service/types";
+import { getServiceDisplayName } from "app/store/service/utils";
+
+type Props = {
+  service: Service;
+};
+
+const ServiceStatus = ({ service }: Props): JSX.Element => {
+  const iconName =
+    service.status === ServiceStatusType.RUNNING
+      ? "success"
+      : service.status === ServiceStatusType.DEAD
+      ? "error"
+      : service.status === ServiceStatusType.DEGRADED
+      ? "warning"
+      : null;
+
+  return (
+    <span>
+      {iconName && (
+        <span className="u-nudge-left--small">
+          <Icon data-testid="service-status-icon" name={iconName} />
```

Comment:
Without an `aria-label` there's no indication what's the status of each service when accessing this via assistive technology.
```suggestion
          <Icon aria-label={iconName} data-testid="service-status-icon" name={iconName} />
```
The above is the easiest way to do it, but would be even better if we used actual `dead`/`degraded`/`running` status.